components/director/internal/open_resource_discovery/validation_error.go

```diff
@@ -0,0 +1,16 @@
+package ord
+
+const (
```

Comment:
we can rename the file and move the "RuntimeError struct" and the "ProcessingError struct" here